src/storage/migrationManager.test.ts

```diff
@@ -25,63 +29,111 @@ describe("storage/migrationManager", () => {
   beforeEach(() => {
     sandbox = sinon.createSandbox();
     executeMigrationsStub = sandbox.stub(migrationUtils, "executeMigrations");
-
-    manager = StorageManager.getInstance();
   });
 
   afterEach(() => {
     sandbox.restore();
   });
 
-  type TestMigrationSetup = [
-    keyof StorageManager,
-    (manager: StorageManager) => Promise<void>,
-    MigrationStorageType,
-    keyof StorageManager,
-  ];
-  const testSetup: TestMigrationSetup[] = [
-    ["getGlobalState", migrateGlobalState, "global", "setGlobalState"],
-    ["getWorkspaceState", migrateWorkspaceState, "workspace", "setWorkspaceState"],
-    ["getSecret", migrateSecretStorage, "secret", "setSecret"],
-  ];
-
-  for (const [
-    managerGetMethodName,
-    migrationFunc,
-    storageType,
-    managerSetMethodName,
-  ] of testSetup) {
-    it(`migrate*() for "${storageType}" state/storage should call executeMigrations() when storage version is incorrect`, async () => {
```

Comment:
This setup was getting a little too complicated; figured it was best to split up into explicit tests for easier maintenance